Backpack/src/main/java/net/skyscanner/backpack/chip/BpkChip.kt

```diff
@@ -18,20 +18,42 @@ open class BpkChip @JvmOverloads constructor(
   defStyleAttr: Int = 0
 ) : BpkText(wrapContextWithDefaults(context), attrs, defStyleAttr) {
 
-  init {
-    initialize(attrs, defStyleAttr)
-  }
-
   var disabled: Boolean = false
     set(value) {
       field = value
       this.isEnabled = !disabled
     }
 
+  var chipBackgroundColor: Int = -1
```

Comment:
just backgroundColor, we don't use prefixes for the code attributes.

Reply:
backgroundColor is already a field/method in View.class. So it cannot and should not be overridden in this way.